<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(125, 183, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelSozdat">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>Создать ваш аккаунт</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSpravka">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Справка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEN">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>EN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRU">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>RU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMake">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>190</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(106, 236, 255);
</string>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2Name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3Password">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(106, 236, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(106, 236, 255);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
